app/tests/test_urls.py

```
@@ -447,15 +447,15 @@ def result_get_data_side_effect(*args, **kwargs):
     @patch("app.context_processors.is_gbb_ended")
-    def test_2025_participants_translation_accessibility(
+    def test_participants_translation_accessibility_current_year(
         self,
         mock_is_gbb_ended,
         mock_get_years,
         mock_context_supabase,
         mock_participants_supabase,
     ):
         """
-        /2025/participants?lang=(すべての言語)にアクセスして200を返すことをテストします。
+        /<lang>/<year>/participants?lang=(すべての言語)にアクセスして200を返すことをテストします（yearは現在年度）。
```

Ruff (0.14.10)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)